docs/internals/howto-release-django.txt

```diff
@@ -337,22 +340,45 @@ You're almost done! All that's left to do now is:
    example, after releasing 1.5.1, update ``VERSION`` to
    ``VERSION = (1, 5, 2, 'alpha', 0)``.
 
-#. For the first beta release of a new version (when we create the
-   ``stable/1.?.x`` git branch), you'll want to create a new
-   ``DocumentRelease`` object in the ``docs.djangoproject.com`` database for
-   the new version's docs, and update the ``docs/fixtures/doc_releases.json``
-   JSON fixture, so people without access to the production DB can still
-   run an up-to-date copy of the docs site.
-
 #. Add the release in `Trac's versions list`_ if necessary (and make it the
    default if it's a final release). Not all versions are declared;
    take example on previous releases.
 
-#. On the master branch, remove the ``UNDER DEVELOPMENT`` header in the notes
-   of the release that's just been pushed out.
-
 .. _Trac's versions list: https://code.djangoproject.com/admin/ticket/versions
 
+New stable branch tasks
+=======================
+
+There are several items to do in the time following a the creation of a new
+stable branch (often following an alpha release). Some of these tasks don't
+need to be done by the releaser.
+
+#. For the first alpha release of a new version (when we create the
+   ``stable/A.B.x`` git branch), create a new ``DocumentRelease`` object in the
```

Comment:
the `For the first alpha release... stable/A.B.x git branch` phrase should probably be moved to the first paragraph.